students/AndrewMiotke/lesson01/assignment/inventory_management/electric_appliance_class.py

```diff
@@ -0,0 +1,37 @@
+""" Electric appliances class """
+
+from inventory_class import Inventory
```

Comment:
These imports need to be modified so that inventory_management is added to the beginning to fix relative imports